addons/website_blog/views/website_blog_templates.xml

```diff
@@ -206,11 +211,15 @@
 <!-- Option: Blog Post List: grid layout -->
 <template id="opt_blog_post_grid_layout" name="Grid view"
         customize_show="True" active="False" inherit_id="website_blog.blog_post_short">
-    <xpath expr="//div[@id='main_column']" position="attributes">
-        <attribute name="class" separator=" " add="d-flex" />
+    <xpath expr="//div[@name='blog_post']" position="replace">
+        <div class="container">
```

Comment:
@kea14 A container cannot be inside of a col.

Bootstrap is: (element with padding or `container`) -> (content or (`row` -> `col`)) -> (content or (`row` -> `col`))  -> ...